packages/edit-site/src/components/sidebar-navigation-screen-patterns/index.js

```diff
@@ -148,28 +140,11 @@ export default function SidebarNavigationScreenPatterns() {
 	return (
 		<SidebarNavigationScreen
 			isRoot={ ! isBlockBasedTheme }
-			title={
-				isTemplatePartsPath
-					? __( 'Manage template parts' )
-					: __( 'Patterns' )
-			}
-			description={
-				isTemplatePartsPath
-					? __(
-							'Create new template parts, or reset any customizations made to the template parts supplied by your theme.'
-					  )
-					: __(
-							'Manage what patterns are available when editing the site.'
-					  )
-			}
-			actions={
-				( isBlockBasedTheme || ! isTemplatePartsPath ) && (
-					<AddNewPattern
-						canCreateParts={ isBlockBasedTheme }
-						canCreatePatterns={ ! isTemplatePartsPath }
-					/>
-				)
-			}
+			title={ __( 'Patterns' ) }
```

Comment:
Just above this return statement, there's a reference to the `/wp_template_part/all` URL we need to update. Hybrid themes no longer have the "Appearance > Template Parts" link in the menu after this PR and we don't load different data depending on the URL either.